ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/DynamoDBKeyValue.java

```diff
@@ -134,6 +154,12 @@ public void init(DataGenerator dataGenerator) throws Exception {
         } else {
             dynamoDB = new DynamoDB(client);
         }
+        autoScalingClient = AWSApplicationAutoScalingClientBuilder.standard()
```

Comment:
I assume we want to configure autoscaling if we use `this.config.programmableTables(`)` otherwise, the user should use the AWS Console to play around with Table configurations. Am I missing something here?

Reply:
yes, programmableTables=false and autoscaling=true is an invalid configuration. ill put this logic in initialize.